### PR TITLE
Adding locopy

### PR DESCRIPTION
**Pull request by fdosani**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jborchma** [comment]
I'm good to be a maintainer.

**Comment by elzzhu** [comment]
Good to be a maintainer.

**Comment by gforsyth** [comment]
Hey @conda-forge/help-python this is ready for review!

File: recipes/locopy/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (dopplershift) [inline recipes/locopy/LICENSE:1]: Looks like `LICENSE` is included in the 0.3.7 tarball, so you can remove from this PR.

File: recipes/locopy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/locopy/meta.yaml:22]: Should also be `python >=3.6`

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor things and this is good to go in.

File: recipes/locopy/LICENSE (hunk 1 above)
Comment (fdosani) [reply recipes/locopy/LICENSE]: Done 👍

File: recipes/locopy/meta.yaml (hunk 2 above)
Comment (fdosani) [reply recipes/locopy/meta.yaml]: Thanks @dopplershift Appreciate it. Should be fixed now.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**